src/features/TimeframeScroll.ts

```diff
@@ -1,27 +1,31 @@
 class TimeframeScroll extends Feature {
+  // Separating hotkeys into individual variables
   triggerDown = false;
-  
+  hotkey = 'Tab';
+  ctrl = false;
+  shift = false;
+  alt = false;
+  meta = false;
```

Comment:
Each of these properties is part of the `Hotkey` interface which gets passed into the `super()` method for classes that extend `Feature`.  Splitting them like this seems redundant, though I agree that inputting an object into the parent class' constructor is tedious.


Side note:
Instead of a string, hotkey should be of type: `Hotkey` which is defined in `src/utils/Hotkey.ts`.